frontend/src/components/MeetingRoom/EmojiGrid/EmojiGrid.tsx

```diff
@@ -29,13 +29,11 @@ const EmojiGrid = ({
 }: EmojiGridProps): ReactElement => {
   const isSmallViewport = useIsSmallViewport();
 
-  const handleClickAway = (event: MouseEvent | TouchEvent) => {
-    const target = event.target as HTMLElement;
-
-    if (isSmallViewport && !target.closest('#emoji-grid-toggle')) {
+  const handleClickAway = () => {
+    if (isSmallViewport) {
       return;
     }
-    // If a user clicks the toggle button, we save their preference for later. If a user doesn't select
+    // If a user clicks the toggle button on mobile, we save their preference for later. If a user doesn't select
```

Comment:
nit I don't think this is really a preference, I think you can just say that this will be handled by the Toolbar.